src/Security/src/Authorization.Certificate/CertificateAuthorizationBuilderExtensions.cs

```diff
@@ -29,13 +29,41 @@ public static class CertificateAuthorizationBuilderExtensions
     /// The incoming <paramref name="builder" /> so that additional calls can be chained.
     /// </returns>
     public static AuthorizationBuilder AddOrgAndSpacePolicies(this AuthorizationBuilder builder)
+    {
+        return AddOrgAndSpacePolicies(builder, null);
+    }
+
+    /// <summary>
+    /// Defines policies that verify the space/org in the incoming client certificate matches the space/org of the local application instance identity
+    /// certificate in configuration.
+    /// <para>
+    /// Secure your endpoints with the included authorization policies by referencing <see cref="CertificateAuthorizationPolicies" />.
+    /// </para>
+    /// <para>
+    /// This method also configures certificate forwarding.
+    /// </para>
+    /// </summary>
+    /// <param name="builder">
+    /// The <see cref="AuthorizationBuilder" /> to configure.
+    /// </param>
+    /// <param name="certificateHeader">
+    /// The name of the header used to pass the certificate.
+    /// </param>
+    /// <returns>
+    /// The incoming <paramref name="builder" /> so that additional calls can be chained.
+    /// </returns>
+    public static AuthorizationBuilder AddOrgAndSpacePolicies(this AuthorizationBuilder builder, string? certificateHeader)
```

Comment:
`certificateHeader` is the name of the HTTP header, right? In that case, a better name would be `certificateHeaderName`.